tests/testers-pipeline.test.sh

```
@@ -0,0 +1,61 @@
+# Integration test for the /uberdev:testers aggregate step.
+# Uses canned per-agent YAML outputs (tests/fixtures/testers-mock-outputs/)
+# to exercise plugins/uberdev/skills/testers-pipeline/aggregate.py
+# deterministically.
+
+set -euo pipefail
+
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+cd "$REPO_ROOT"
+
+SCRATCH="$(mktemp -d -t testers-pipeline-XXXX)"
+trap "rm -rf $SCRATCH" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)